sdk/typescript/README.md

```diff
@@ -40,6 +40,74 @@ const gatewayApi = GatewayApiClient.initialize({
 const { status, transaction, stream, state } = gatewayApi
 ```
 
+## Connecting to a secured gateway using authorization header
+
+This example shows how to provide basic auth, JWT and Bearer Token can be done the same way by passing the authorization header.
+
+To work with this set-up you will need to:
+* Ensure your `basePath` is set up to connect to the correct host and port, on which Gateway API. is exposed.
+* Set the `Authorization` header on your request, configured with your basic auth credentials for the Gateway API.
+* In the `agent` / `dispatcher` configuration, include the self-signed certificate by using the `ca` parameter, or if you understand the implications and have precautions against MITM, use `rejectUnauthorized: false` with the https agent.
+
+### With `node-fetch`
+
+An example, with the `rejectUnauthorized: false` line, assuming you are EG on localhost, or a private network where MITM attacks are mitigated.
```

Comment:
I would get rid of this line. Generally Gateways will have valid certificates (unlike nodes)

Reply:
fixed